TESReloaded/Core/RenderManager.cpp

```diff
@@ -157,21 +172,21 @@ void RenderManager::SetupSceneCamera() {
 
 		InvViewMatrix = invViewMatrix;
 
-		projMatrix._11 = 2.0f / RmL;
+		projMatrix._11 = 2.0f / FrustumWidth;
 		projMatrix._12 = 0.0f;
 		projMatrix._13 = 0.0f;
 		projMatrix._14 = 0.0f;
 		projMatrix._21 = 0.0f;
-		projMatrix._22 = 2.0f / TmB;
+		projMatrix._22 = 2.0f / FrustumHeight;
 		projMatrix._23 = 0.0f;
 		projMatrix._24 = 0.0f;
-		projMatrix._31 = -RpL / RmL;
-		projMatrix._32 = -TpB / TmB;
-		projMatrix._33 = Frustum->Far * InvFmN;
+		projMatrix._31 = -RpL / FrustumWidth;
+		projMatrix._32 = -TpB / FrustumHeight;
+		projMatrix._33 = Q;

```

Comment:
What's  changing projMatrix._33 from Frustum->Far * InvFmN to farZ / (farZ - nearZ) do?
Also the cascaded changes to projMatrix._43

Reply:
it's the same thing, just less obscured and closer in formulation to the type of syntax that can be found in ressources on the subject on the net